.github/workflows/conformance-datapath.yaml

```diff
@@ -218,8 +218,10 @@ jobs:
             --nodes-without-cilium=kind-worker3 --helm-set-string="kubeProxyReplacement=${{ env.KPR }}"
 
             ./cilium-cli status --wait
-            ./cilium-cli connectivity test --datapath
-            ./cilium-cli connectivity test -t -d
+            ./cilium-cli connectivity test --datapath --collect-sysdump-on-failure \
+              --sysdump-output-filename cilium-sysdump-${{ matrix.kernel }}-<ts>
+            ./cilium-cli connectivity test -t -d --collect-sysdump-on-failure \
+              --sysdump-output-filename cilium-sysdump-${{ matrix.kernel }}-<ts>
```

Comment:
I see we are using the same filename: is the connectivity test smart enough not to overwrite the existing sysdump?

Reply:
The filenames will include a timestamp (the `<ts>` part) so they won't be the same.

Reply:
However, we were missing double quotes here :grimacing: Fixed in the latest push.